site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -229,6 +229,8 @@ The following tables lists the configurable parameters of the Agones chart and t
 | Parameter                                           | Description                                                                                     | Default                |
 | --------------------------------------------------- | ----------------------------------------------------------------------------------------------- | ---------------------- |
 | `agones.allocator.http.loadBalancerIP`              | The [Load Balancer IP][loadBalancerIP] of the Agones allocator load balancer. Only works if the Kubernetes provider supports this option. | ""                     |
+| `agones.allocator.disableMTLS`                      | Turns off client cert authentication for incoming connections to the allocator.            | `false`                |
```

Comment:
One of this option was present, not sure if the functionality was the same as current one.

Reply:
There is a breaking change for `disableMTLS` between 1.8 and 1.9. The helm configuration was not documented because we did not want anyone get a dependency to it, until we know exactly what are the requirements for GKE ingress. If it is moved to 1.8 documentation, a user may get a dependency and the behavior changes with the next version.

Reply:
Even though it wasn't documented, should we add a 1.9 release note that mentions the breaking change?

Reply:
I don't think we have to document it as a breaking change as it has not been documented at all and off by default. Only by reading the code the feature flag could be discovered. However, if there is a best practices on what to do in this situation, we should go with it. Otherwise, I would say it is unnecessary.

Reply:
+1 to what Pooneh mentioned. disableMTLS was never documented earlier.

Reply:
Just confirming before merging this - it is good to merge?